app/components/CharacterSelect.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import Relay from 'react-relay';
+
+import CreateCharacterMutation from '../mutations/CreateCharacterMutation';
+
+/**
+ * The button to display or hide the character creation form.
+ * @param {type} props
+ * @returns {Function|String}
+ */
+function CreationButton(props) {
+    if (props.toggle) {
+        return (
+            <div className="daenerys-add" onClick={() => {props.onClick();}}>^</div>
+        );
+    } else {
+        return (
+            <div className="daenerys-add" onClick={() => {props.onClick();}}>+</div>
```

Comment:
I would clean this up with something like 
```
let content = props.toggle ? '^' : '+';
return (
  <div className="daenerys-add" onClick={() => {props.onClick();}}>{{content}}</div>
);
```

Not sure if that's the exact syntax for printing a variable's content.